src/components/buttons/critical-action-button/critical-action-button.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import filterAriaAttributes from '../../../utils/filter-aria-attributes';
+import Spacings from '../../spacings';
+import styles from './critical-action-button.mod.css';
+
+const getStateClassName = ({ isToggled, isDisabled }) => {
+  if (isDisabled) return styles.disabled;
+  if (isToggled) return styles.active;
+  return styles.button;
+};
+const getSizeClassName = size => styles[size];
+
+const CriticalActionButton = props => {
+  const dataProps = {
+    'data-track-component': 'CriticalActionButton',
+    ...filterDataAttributes(props),
+  };
+  const ariaProps = {
+    // TODO: Turn `aria-label` into its own prop and make it required
+    // in order to make the component more accessible. Currently, for backwards
+    // compatibility reasons, `aria-label` contains the `label` prop value.
+    'aria-label': props.label,
```

Comment:
Exactly. We want to keep the "accessibility" for this component. It means that by default (if the prop `props['aria-label']` is not provided) it will take `props.label` as default.